model/events/LtiAgsListener.php

```diff
@@ -117,6 +119,8 @@ private function onDeliveryExecutionFinish(DeliveryExecutionState $event): void
             /** @var QueueDispatcherInterface $taskQueue */
             $taskQueue = $this->getServiceLocator()->get(QueueDispatcherInterface::SERVICE_ID);
             $taskQueue->createTask(new SendAgsScoreTask(), [
+                SendAgsScoreTask::RETRY_MAX => $this->getAgsMaxRetries(),
+                SendAgsScoreTask::RETRY_COUNT => 0,
```

Comment:
Providing an initial retry count looks a bit silly, don't you think?
```suggestion
```
Let's remove it.